strictdoc/server/helpers/turbo.py

```diff
@@ -9,4 +12,4 @@ def render_turbo_stream(content: str, action: str, target: str):
   </template>
 </turbo-stream>
 """
-    return turbo_stream
+    return Markup(turbo_stream)
```

Comment:
Related to my other comment's question: Does this mean that the Markup will be applied two times: one time here and one time if the `content` is rendered with `render_template_as_markup`?

Reply:
No, it's explained in [markupsafe.Markup](https://markupsafe.palletsprojects.com/en/latest/escaping/#markupsafe.Markup):

> Passing an object to the constructor converts it to text and wraps it to mark it safe *without escaping*.